usbinfo/__main__.py

```diff
@@ -178,7 +190,7 @@ def __init__(self):
 
 
 def main(args=None):
-    """Main function"""
+    """Main function."""
```

Comment:
Can take this comment out.

Reply:
Probably will leave this in for style-guide conformity.